api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
Or I can try updating exec to accept both
```
exec?command=echo&command=ABC
``` 
and
```
exec?command=echo,ABC
```
(the second case will not work for commands with ',' tough)